addons/website_event_questions/views/event_templates.xml

```diff
@@ -31,16 +31,19 @@
 </template>
 
 <template id="registration_event_question" name="Registration Event Question">
-    <label t-esc="question.title"/>
+    <label t-esc="question.title"/> <span t-if="question.mandatory_answer" class="mandatory_mark"> *</span>
```

Comment:
Note that span could be added at next line

Reply:
Done and noted, thanks